group_vars/prosody_staging/vars.yml

```diff
@@ -1,10 +1,10 @@
 ###
-# QA specific variables for ppa-django
+# Staging specific configuration for Prosody Archive (ppa-django)
 ###
 ---
 
 # email prefix for dev account emails
-email_prefix: '[PPA QA] '
+email_prefix: '[TEST PPA] '
```

Comment:
I don't entirely follow what this is for, but it's the only place in this PR where you're changing `QA` to `TEST`. Should it be `STAGING` to match everything else?

Reply:
yes, it probably should! this is the subject prefix if we get error emails from the app (which I'm not sure we actually have set up). I like the brevity of qa or test but you're right it's inconsistent. I'll update all the app email prefixes.